src/WinRT.Runtime/ApiCompatBaseline.net5.0.txt

```diff
@@ -1,4 +1,5 @@
 Compat issues with assembly WinRT.Runtime:
 MembersMustExist : Member 'public WinRT.MarshalString.HStringHeader WinRT.MarshalString.HStringHeader WinRT.MarshalString._header' does not exist in the implementation but it does exist in the contract.
 TypesMustExist : Type 'WinRT.MarshalString.HStringHeader' does not exist in the implementation but it does exist in the contract.
-Total Issues: 2
+MembersMustExist : Member 'public void WinRT.SingleInterfaceOptimizedObject..ctor(System.Type, WinRT.IObjectReference)' does not exist in the implementation but it does exist in the contract.

```

Comment:
This is API compat breaking, we should look at adding multiple constructors instead rather than modifying existing one.